ui/pages/confirmations/confirm/stories/signatures/permit.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import ConfirmPage from '../../confirm';
+import { permitSignatureMsg } from '../../../../../../test/data/confirmations/typed_sign';
+import {
+  ARG_TYPES_SIGNATURE,
+  CONFIRM_PAGE_DECORATOR,
+  SignatureStoryTemplate,
+} from './utils';
+
+/**
+ * The `<ConfirmPage>` that's displayed when the current confirmation is either a
+ * "V3" or "V4" `eth_signTypedData` signature that parses as a valid permit signature.
+ */
+export default {
+  title: 'Pages/Confirmation/ConfirmPage/Signatures/Permit',
```

Comment:
To match the file path and component names, should this be `Pages/Confirmations/Confirm/Signatures/Permit`?

Reply:
whoops. updated  [1298c2b](https://github.com/MetaMask/metamask-extension/pull/25283/commits/1298c2bd8b80586a45e725a042257fbe04277698) and updated  ConfirmationNetworkSwitch path 9158977